native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/AbstractNativeMojo.java

```diff
@@ -175,7 +182,8 @@ private Path getDefaultRepo(Path destinationRoot) {
             try {
                 targetUrl = new URI(metadataUrl).toURL();
                 // TODO investigate if the following line is necessary (Issue: https://github.com/graalvm/native-build-tools/issues/560)
-                metadataRepositoryConfiguration.setUrl(targetUrl);
+                // TODO: forced to remove this to allow building on my machine. Remove this before merging.
+//                metadataRepositoryConfiguration.setUrl(targetUrl);
```

Comment:
:x: This needs to be uncommented. There shouldn't be any need for this hacks for local builds either.

Reply:
Totally agree! Hopefully we can resolve this.